server/models/db/student.js

```diff
@@ -10,7 +10,21 @@ const Tables = require('../../constants/tables.json');
 async function getStudent(email, password) {
     return userModel.getUser(email, password, Tables.student, Student);
 }
+// This method inserts a student account given specific information.
+async function insertStudentAccount(firstName, lastName, type, studentID, email, password, phone, school_id ) {
```

Comment:
Hi Nancy,
To keep it consistent, I would recommend adjusting the naming of `school_id` to `schoolId` (lowerCamelCase)

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've changed the name from school_ID to schoolId.
You can now see these changes the following locations:

line 14 of server/models/db>student.js
line 18 of server/models/db>student.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy,
Awesome! Thank you for making those changes.